build/scripts/Releasing.fsx

```diff
@@ -34,10 +34,8 @@ module Release =
 
             let year = sprintf "%i" DateTime.UtcNow.Year
 
-            let jsonDotNetVersion = 10
-
-            let jsonDotNetCurrentVersion = sprintf "%i" jsonDotNetVersion
-            let jsonDotNetNextVersion = sprintf "%i" (jsonDotNetVersion + 1)
+            let jsonDotNetCurrentVersion = "10.0.1"
```

Comment:
It would be awesome if this value were retrieved from the `<PackageReference Include="Newtonsoft.Json" Version="10.0.1" />` in Nest.csproj. Using an XML type provider (or even just LINQ to XML), could grab the current version from thecsproj. Then, using `SemVerHelper` from F# FAKE, can parse the semantic version to get the next major version for `jsonDotNetNextVersion` .

Reply:
OK cool, I'll have a go at that and see how it looks

Reply:
Awesome 😃 ping me if you have questions